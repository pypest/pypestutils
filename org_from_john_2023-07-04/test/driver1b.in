22
pt01.hds
1
driver1b.out


