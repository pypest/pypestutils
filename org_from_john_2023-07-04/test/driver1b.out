     KSTP      KPER    PERTIM            TOTIM                       TEXT       NSTRT      NVAL      ILAY
        1         1    1.00000000        1.00000000                 HEADU           1      6840         1
        1         1    1.00000000        1.00000000                 HEADU        6841     13680         2
        1         2    219.000000        220.000000                 HEADU           1      6840         1
        1         2    219.000000        220.000000                 HEADU        6841     13680         2
        2         2    438.000000        439.000000                 HEADU           1      6840         1
        2         2    438.000000        439.000000                 HEADU        6841     13680         2
        3         2    657.000000        658.000000                 HEADU           1      6840         1
        3         2    657.000000        658.000000                 HEADU        6841     13680         2
        4         2    876.000000        877.000000                 HEADU           1      6840         1
        4         2    876.000000        877.000000                 HEADU        6841     13680         2
        5         2    1095.00000        1096.00000                 HEADU           1      6840         1
        5         2    1095.00000        1096.00000                 HEADU        6841     13680         2
        1         3    365.000000        1461.00000                 HEADU           1      6840         1
        1         3    365.000000        1461.00000                 HEADU        6841     13680         2
        2         3    730.000000        1826.00000                 HEADU           1      6840         1
        2         3    730.000000        1826.00000                 HEADU        6841     13680         2
        3         3    1095.00000        2191.00000                 HEADU           1      6840         1
        3         3    1095.00000        2191.00000                 HEADU        6841     13680         2
        4         3    1460.00000        2556.00000                 HEADU           1      6840         1
        4         3    1460.00000        2556.00000                 HEADU        6841     13680         2
        5         3    1825.00000        2921.00000                 HEADU           1      6840         1
        5         3    1825.00000        2921.00000                 HEADU        6841     13680         2
        1         4    2190.00000        5111.00000                 HEADU           1      6840         1
        1         4    2190.00000        5111.00000                 HEADU        6841     13680         2
        2         4    4380.00000        7301.00000                 HEADU           1      6840         1
        2         4    4380.00000        7301.00000                 HEADU        6841     13680         2
        3         4    6570.00000        9491.00000                 HEADU           1      6840         1
        3         4    6570.00000        9491.00000                 HEADU        6841     13680         2
        4         4    8760.00000        11681.0000                 HEADU           1      6840         1
        4         4    8760.00000        11681.0000                 HEADU        6841     13680         2
        5         4    10950.0000        13871.0000                 HEADU           1      6840         1
        5         4    10950.0000        13871.0000                 HEADU        6841     13680         2
